server/fishtest/schemas.py

```diff
@@ -422,6 +422,8 @@ def final_results_must_match(run):
                 "threads": suint,
                 "resolved_base": sha,
                 "resolved_new": sha,
+                "master_sha?": sha,
+                "official_master_sha?": sha,
```

Comment:
Will master_sha and official_master_sha always be present in new runs? In that case there is no need to make them optional. If they are only for the SPSA tests then things are different of course.

Reply:
> Will master_sha and official_master_sha always be present in new runs? In that case there is no need to make them optional. If they are only for the SPSA tests then things are different of course.

Removed optional and fixed test cases.

Reply:
that being said I think a type of `any` might be useful in the future.

Reply:
The type “object” plays the role of “any”. I will put that in the readme.